pkgpanda/build/cli.py

```diff
@@ -23,11 +23,18 @@ def main():
     try:
         arguments = docopt(__doc__, version="mkpanda {}".format(pkgpanda.build.constants.version))
         umask(0o022)
-
+        variant_arg = arguments['--variant']
+        # map the keyword 'default' to None to build default, but use the null arguments
+        # to trigger all variants
+        target_variant = variant_arg if variant_arg != 'default' else None
         # Make a local repository for build dependencies
         if arguments['tree']:
             package_store = pkgpanda.build.PackageStore(getcwd(), arguments['--repository-url'])
-            pkgpanda.build.build_tree(package_store, arguments['--mkbootstrap'], arguments['<variant>'])
+            if variant_arg is None:
+                for variant in pkgpanda.build.get_variants_from_filesystem(getcwd(), 'treeinfo.json'):
+                    pkgpanda.build.build_tree(package_store, arguments['--mkbootstrap'], variant)
```

Comment:
The `get_variants_from_filesystem()` call feels overly detailed for a CLI script's main function. Let's stuff this loop into a `build_tree_variants()` helper, similar to `build_package_variants()`.